src/istgtcontrol.c

```diff
@@ -1400,6 +1435,7 @@ static EXEC_TABLE exec_table[] =
 	{"RSV", exec_rsv, 0, 0 },
 	{"QUE", exec_que, 0, 0 },
 	{"STATS", exec_stats, 0, 0 },
+	{"IOSTATS", exec_iostats, 0, 0 },
```

Comment:
`IOSTATS` is for replication feature only. If Replication is disabled then the target will return an error. So better to add macro protection here if possible.